robot_skills/src/robot_skills/base.py

```diff
@@ -134,8 +134,9 @@ def getPlan(self, position_constraint):
     def checkPlan(self, plan):
         try:
             resp = self._check_plan_client(plan)
-        except:
+        except Exception as e:
             rospy.logerr("Could not check plan, is the global planner running?")
+            rospy.logerr(e)
```

Comment:
Why don't you do it in a single log statement:
```python
rospy.logerr("Could not check plan: {}".format(e.message))
```
That would result in a sinlge logrecord of an exception instead of multiple ones. This is much clearer in e.g., rqt-console (and even the terminal: less clutter by timestamps and levels)